Source/Shared/Paths.cs

```diff
@@ -78,23 +78,33 @@ private static string GetAppBaseDir()
     public static readonly string BundledResourceDir = ContentDirPath;
 
     /// <summary>Directory for the downloaded resources. Read + write access.</summary>
-    // TODO[#93]: Better download path, like the user Downloads folder or AppData
-    public static readonly string DownloadedResourceDir = ContentDirPath;
+    public static readonly string DownloadedResourceDir = Directory.CreateDirectory(
+        Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+            "Bloodmasters"
+        )).FullName;
```

Comment:
Let's add `Downloads` subfolder there? I don't want everything to stay in the same folder.